src/types/records-types.ts

```diff
@@ -83,6 +83,7 @@ export type UnsignedRecordsWriteMessage = {
  */
 export type RecordsQueryReplyEntry = RecordsWriteMessage & {
   encodedData?: string;
+  messageCid: string;
```

Comment:
Design: I think we should have the cursor value outside of the array of entries, as metadata to be used for subsequent query.

Reply:
I think if we do land on a more general 'cursor' type from the above conversation, this is the right move.

But if we stay with specifically `messageCid` as the cursor, I think it might be a good idea to keep this here.

I might want to fetch the page prior to what I have again (sort the opposite direction with the first record's cursor).

Reply:
Upon further discussion, agree with you that `messageCid` as the cursor has the better trade offs.